dm/worker/relay.go

```diff
@@ -234,6 +234,11 @@ func (h *realRelayHolder) stopRelay(ctx context.Context, op pb.RelayOp) error {
 	h.stage = pb.Stage_Stopped
 	h.Unlock() // unlock to make `run` can return
 
+	// purge relay dir when delete source
+	if err := h.relay.PurgeRelayDir(); err != nil {
+		return err
+	}
+
```

Comment:
how to change 🤔

Reply:
I suggest moving relay dir to another name instead of deleting it directly. We have `purge-relay` command and I think the purge action should only happen in this command.

Reply:
`relay_dir_deprecated` is out of `relay_dir` thus can't be deleted using `purge-relay`. And disk space is not freed

Reply:
I don't think it's a good idea. `stop source` is a normal operation. shouldn't purge data or move to another directory.